### PR TITLE
Feat/packer v2 gh runner img

### PR DESCRIPTION
**Pull request by DenisBorzenkov**
gh-runner pipeline

### CODE REVIEW

File: .github/workflows/packer-gh-runner-img.yml [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+          make validate
+
+      - name: Install dependencies
+        run: pip install pre-commit
+
+      - name: Run pre-commit
+        run: pre-commit run --all-files
+
+  Build:
+    runs-on: packer-runner
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/packer-gh-runner-img.yml]: this needs to be a self-hosted runner

**Review by AlexFernandes-MOVAI — COMMENTED** [review]
Please review the steps for :
- build on push: local build only
- staging on merge to main: build and store on some place that can be reused STAGING_DIR
- deployment on release of official version:  read the artifacts from STAGING_DIR to copy them to NAS

File: .github/workflows/packer-workflow-img.yml [hunk 2]
```diff
@@ -45,11 +41,9 @@ on:
         required: true
 
 env:
-  MINIO_S3_URL: "https://s3.mov.ai"
-  MINIO_PUBLIC_URL: "https://minio.mov.ai"
   RUN_STAGING_DIR: "/home/devops/packer-ci/staging/${{ github.repository }}/${{ github.run_id}}/"
   PROXMOX_NFS_SERVER_PATH: "/media/nas-mary/template/iso"
-  PROXMOX_NFS_SERVER_URL: "https://mary.mov.ai:8006/#v1:0:=storage%2Fbb-8%2Fnas-mary:4:5:=contentIso"
+  PROXMOX_NFS_SERVER_URL: "https://hel.mov.ai:8006/#v1:0:=storage%2Fbb-8%2Fnas-mary:4:5:=contentIso"
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/packer-workflow-img.yml]:
```suggestion
  PROXMOX_NFS_SERVER_URL: "https://hel.mov.ai:8006/#v1:0:=storage%2Fhel%2Fnas-mary:4:5:=contentIso:::::"
```

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]
just a change  on the proxmox URL

File: .github/workflows/packer-gh-runner-img.yml (hunk 1 above)
Comment (DenisBorzenkov) [reply .github/workflows/packer-gh-runner-img.yml]: done

**Pull request merged**